### ISSUE TITLE
`helm init` failed if my client cannot access network

### ISSUE THREAD

**Issue by gyliu513** [kubernetes/helm#2864]
```
# /helm search
Error: open /root/.helm/repository/repositories.yaml: no such file or directory
# /helm init -c
Creating /root/.helm
Creating /root/.helm/repository
Creating /root/.helm/repository/cache
Creating /root/.helm/repository/local
Creating /root/.helm/plugins
Creating /root/.helm/starters
Creating /root/.helm/cache/archive
Creating /root/.helm/repository/repositories.yaml
Error: Looks like "https://kubernetes-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: Get https://kubernetes-charts.storage.googleapis.com/index.yaml: x509: failed to load system roots and no roots provided
```

In a client host which cannot access network, the `helm init -c` will always failed, this is not acceptable, as many people want to run `helm init -c` without internet access.

The proposal is that we can introduce a new parameter to skip the stable repo from google. such as `helm init -c --skip-stable-repo`

**Comment by seh** [comment]
It is hard to say whether this is a bug or a desired feature that's absent. What was the design intention of _init -c_?

**Comment by gyliu513** [comment]
`init -c` is mainly used for config the helm client for end user, so that they can use `helm CLI`. @seh 

For this request, I want to make sure the `helm CLI` can also works even cannot access internet.

**Comment by bacongobbler** [comment]
Closing as a duplicate of https://github.com/kubernetes/helm/issues/2235. Use the `--skip-refresh` flag, which has been available since v2.3.0. Thanks!
